bucket4j/src/main/java/com/linecorp/armeria/server/throttling/bucket4j/TokenBucketThrottlingStrategyBuilder.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.throttling.bucket4j;
+
+import java.time.Duration;
+import java.util.Objects;
+
+import javax.annotation.Nullable;
+
+import com.linecorp.armeria.common.Request;
+import com.linecorp.armeria.common.throttling.ThrottlingHeaders;
+
+/**
+ * Builds {@link TokenBucketThrottlingStrategy}.
+ */
+public final class TokenBucketThrottlingStrategyBuilder<T extends Request> {
+
+    private final TokenBucket tokenBucket;
+    @Nullable
+    private Duration minimumBackoff;
+    @Nullable
+    private ThrottlingHeaders headersScheme;
+    private boolean sendQuota;
+    @Nullable
+    private String name;
+
+    TokenBucketThrottlingStrategyBuilder(TokenBucket tokenBucket) {
+        this.tokenBucket = Objects.requireNonNull(tokenBucket, "tokenBucket");
+    }
+
+    /**
+     * Optional name of the strategy.
+     * By default, it will be assigned with a predefined name.
+     */
+    public TokenBucketThrottlingStrategyBuilder<T> name(String name) {
+        this.name = name;
```

Comment:
Could you add null check even though the `name` is nullable?

Reply:
Fixed